contracts/TopShot.cdc

```diff
@@ -380,6 +381,45 @@ pub contract TopShot: NonFungibleToken {
             return <-newCollection
         }
 
+        pub fun mintMomentWithSubedition(playID: UInt32, subeditionID: UInt32): @NFT {
+            pre {
+                self.retired[playID] != nil: "Cannot mint the moment: This play doesn't exist."
+                !self.retired[playID]!: "Cannot mint the moment from this play: This play has been retired."
+            }
+
+            // Gets the number of Moments that have been minted for this Play
+            // to use as this Moment's serial number
+            let numInSubedition = TopShotRemix.getNumberMintedPerSubedition(setID: self.setID,
+                                                                            playID: playID,
+                                                                            subeditionID: subeditionID)
+
+            // Mint the new moment
+            let newMoment: @NFT <- create NFT(serialNumber: numInSubedition + UInt32(1),
+                                              playID: playID,
+                                              setID: self.setID)
+
```

Comment:
We will also need to have a way to know which subedition an NFT is a member of. Let's talk through our options for this.

Reply:
Add map NFT->subedition to TopShotRemix contract?